NPS_generation/commands/create_training_sets.py

```diff
@@ -48,7 +47,7 @@ def add_args(parser):
         "--max-tries", type=int, default=200, help="Maximum tries to get n_molecules with min_tc"
     )
     parser.add_argument(
-        "--seed", type=int, default=None, help="Random Seed"
+        "--seed", type=seed_type, nargs="?", default=None, help="Random Seed"
```

Comment:
A utility function `seed_type` is used since snakemake passes the flag as `--seed None`, so we need to be able to interpret the string `"None"` as `None`.